web_src/js/components/RepoContributors.vue

```diff
@@ -21,6 +21,24 @@ import $ from 'jquery';
 
 const {pageData} = window.config;
 
+const colors = {
+  text: '--color-text',
+  border: '--color-secondary-alpha-60',
+  commits: '--color-primary-alpha-60',
+  additions: '--color-green',
+  deletions: '--color-red',
```

Comment:
@sahinakkaya Are these colors unused?

Reply:
I can't see the effect of the changes in frontend part :( I don't understand what you mean unfortunately.

Reply:
I mean we assign green/red colors but from what I can tell the charts always render in the "commits" color for me. I was wondering where red/green is supposed to show up.

Regarding changes not showing, try another browser as well and a `make clean-all`.

Reply:
Oh I see. The other colors are supposed to show when you select other contribution types from select box at top. 
![image](https://github.com/go-gitea/gitea/assets/32161460/ae1f945d-93b1-4c6e-8db3-b92456df2cb5)

Reply:
Ah, I totally missed that this box is clickable.